app/services/courses_query.rb

```diff
@@ -5,10 +5,11 @@ def self.call(params = {})
     new(params:).call
   end
 
-  attr_reader :scope
+  attr_reader :scope, :params
 
   def initialize(params:)
```

Comment:
I've just realised, the `.call` argument signature differs from the `#initialize` signature.

I think there's more value in consistency.

Reply:
I didn't understood.

```ruby
 3 class CoursesQuery
   4   def self.call(params = {})
   5     new(params:).call
   6   end
   9
  10   def initialize(params:)
```

They have the same signature. Do you meant something else?

Reply:
```ruby
CoursesQuery.call(params) # accepts positional argument
CoursesQuery.new(params: params).call # accepts keyword argument
```

I'd do the following to keep them consistent:

```ruby
def self.call(...)
  new(...).call # pass everything received in .call to .new
end
```

Reply:
I see! Yes I will do.